nexx_integration.module

```diff
@@ -16,10 +16,10 @@ use Drupal\media_entity\Entity\Media;
 function nexx_integration_taxonomy_term_insert(EntityInterface $term) {
   $response_data = nexx_integration_taxonomy_term_request($term, 'insert');
 
-  if (!empty($response_data['item'])) {
+  if (!empty($response_data['result'])) {
```

Comment:
Maybe check for `$response_data['result']['itemupdate']['generatedID']` instead, as this entry will be used then.